src/lib/nvme_pt_ioctl.c

```diff
@@ -1847,8 +1842,9 @@ int opal_lock_unlock_pt(struct sed_device *dev, const struct sed_key *key,
 {
 	int ret = 0;
 	struct opal_device *opal_dev;
+	uint32_t lk_type = lock_type;
```

Comment:
why do we need to copy lock_type to  lk_type variable? can we use lock_type instead

Reply:
The reason for assigning it to a uint32_t type is that we needn't have to check for the negative value being passed. Only one check condition is sufficient.

Reply:
ok, makes sense